lib/rust/ensogl/core/src/display/navigation/navigator/events.rs

```diff
@@ -3,7 +3,7 @@ use crate::prelude::*;
 use crate::control::callback;
 use crate::control::io::mouse;
 use crate::control::io::mouse::MouseManager;
-use crate::display::navigation::navigator::SharedSwitch;
+use crate::display::navigation::navigator::NavigatorSettings;
```

Comment:
`navigator::NavigatorSettings` – again, the name `navigator` repeats here. Prefer qualified imports, always.

Reply:
Done